### ISSUE TITLE
Code Building Blocks Don't Work

### ISSUE THREAD

**Issue by judy2k** [judy2k/doc-to-md#1]
Code building blocks export to HTML in the worst way. 

```html
<p class="c0"><span class="c7">bigframes.options.bigquery.project = </span><span class="c4">&quot;GCP project
            ID&quot;</span></p>
    <p class="c0 c13"><span class="c5"></span></p>
    <p class="c0"><span class="c7">#</span><span class="c14 c3">&nbsp;</span><span class="c7">df</span><span
            class="c14 c3">&nbsp;= &lt;pandas_df&gt;</span></p>
    <p class="c0"><span class="c7">bdf = bpd.read_pandas(df)</span></p>
```

* The top-level "c0" paragraphs (one per line) do not specify a typeface, so they can't be identified as code.
* The individual, syntax-highlighted tokens within the paragraphs *are* marked up with a fixed-width font (Roboto Mono), but this means the spaces between the tokens default to the document's default typeface.
* This makes it look like a paragraph with lots of code literal spans in it - which is how doc2md renders it in Markdown.

# Expected behaviour

Code building blocks should be recognised &mdash; somehow &mdash; then all of the spans should be flattened out, and the contiguous code paragraphs (because each line is a paragraph) should be merged into a code block.

**Comment by judy2k** [comment]
Hopefully this should be fixed by implementing #3.

**Comment by judy2k** [comment]
I discovered there are some spans that mark the start and end of code building blocks, so the fix for this (now implemented) should be relatively robust.
